Assets/Scripts/CopilotMechanics/MushroomFriendPassive.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MushroomFriendPassive : CopilotPassiveMechanic
+{
+    public float defenseIncrease; //amount to increase defense
+    public float sporeTimer; //lifetime of spores
+    private int healthLost; //set threshold amount of 10% health decrease
+    private int health; //to detect if health decreased or increased
+    void Start()
+    {
+        health = PlayerInfo.singleton.health;
+        healthLost = (int)(PlayerInfo.singleton.health * 0.1f);
+        PlayerInfo.singleton.damageEvent += AddSpore;
+    }
+    public void AddSpore()
+    {
+        if ((health - PlayerInfo.singleton.health) >= healthLost)
+        {
+            health = PlayerInfo.singleton.health;
```

Comment:
You might need to clarify this more with the design team but this doesn't account for when the player is healed. So for instance if I were to take 5% damage but then got healed to full then does the passive keep track of that previous 5% or not?